commit 01f06c30857b7ef5540425b12cc381057f0b057c
Author: Thomas Lange <lange@informatik.uni-koeln.de>
Date:   Mon Jan 2 14:32:14 2017 +0100

    add -p to grub-mkimage call

diff --git a/bin/fai-cd b/bin/fai-cd
index 03f4714..b65551f 100755
--- a/bin/fai-cd
+++ b/bin/fai-cd
@@ -166,7 +166,7 @@ create_grub2_image() {
         cp $NFSROOT/boot/grub/core.img $tmp/boot/grub
         MODULEBASE=$NFSROOT/usr/lib/grub
     elif [ -f $NFSROOT/usr/lib/grub/i386-pc/cdboot.img ]; then
-        chroot $NFSROOT /usr/bin/grub-mkimage -d /usr/lib/grub/i386-pc -o /boot/grub/core.img biosdisk iso9660 --format=i386-pc
+        chroot $NFSROOT /usr/bin/grub-mkimage -p /boot/grub -d /usr/lib/grub/i386-pc -o /boot/grub/core.img biosdisk iso9660 --format=i386-pc
         cp $NFSROOT/boot/grub/core.img $tmp/boot/grub
         MODULEBASE=$NFSROOT/usr/lib/grub
     else
@@ -180,7 +180,7 @@ create_grub2_image() {
         mkdir -p $GRUBDIR
         dpkg -x $grubPackage $GRUBDIR
         cd $GRUBDIR
-        ./usr/bin/grub-mkimage -d usr/lib/grub/*-pc -o $tmp/boot/grub/core.img biosdisk iso9660 --format=i386-pc
+        ./usr/bin/grub-mkimage -p /boot/grub -d usr/lib/grub/*-pc -o $tmp/boot/grub/core.img biosdisk iso9660 --format=i386-pc
         MODULEBASE=$GRUBDIR/usr/lib/grub
     fi
 
